src/config.rs

```diff
@@ -94,6 +107,12 @@ impl Config {
     pub fn crypto_provider(&self) -> &CryptoProvider {
         &self.crypto_provider
     }
+
+    /// The policy the client and server will follow for use_srtp extension (rfc5764).
+    #[inline(always)]
+    pub fn use_srtp(&self) -> ExtensionPolicy {
+        self.use_srtp.clone()
```

Comment:
The `ExtensionPolicy` enum is so simple it should be a `Copy` type, which means we don't need to clone here.